test/app/teams_test.rb

```diff
@@ -91,6 +91,17 @@ def test_user_must_be_manager
     end
   end
 
+  def test_logged_in_user_can_search_teams
+    [
+      [:get, '/teams'],
+      [:get, '/teams/']
+    ].each do |verb, endpoint|
```

Comment:
Yeah. I was aware of that, but since all the tests here use the same approach, I did not have any strong opinions and made it consistent with others. 

Writing separate tests would add numerous tests which would be slightly out of context in this file. Other way would be refactor the entire file and split it into multiple files.